styles/style.css

```diff
@@ -24,7 +34,69 @@ hr {
   text-align: center;
 }
 .text_offer {
+  font-family: Roboto;
+  font-style: normal;
   padding-top: 48px;
   font-size: 24px;
   color: #383838;
 }
+/*Footer styles*/
+.footer {
```

Comment:
and here

.footer {
  max-width: 1440px;
  width: 100%;
  margin: 0 auto;
  background-color: #10367c;
  display: flex;
  justify-content: normal;
  flex-wrap: wrap;
}